app/Http/Controllers/Staff/DonationController.php

```diff
@@ -74,16 +74,31 @@ public function update(Request $request, int $id): \Illuminate\Http\RedirectResp
         $donation->status = ModerationStatus::APPROVED;
         $donation->starts_at = $now;
```

Comment:
```suggestion
```

Instead of setting it here and overwriting it in specific cases later, it would be good if it was explicitly set inside each edge case to make things more clear.